pkg/amazon/backend/up.go

```diff
@@ -16,14 +16,14 @@ func (b *Backend) Up(ctx context.Context, options cli.ProjectOptions) error {
 		return err
 	}
 
-	if b.Cluster != "" {
-		ok, err := b.api.ClusterExists(ctx, b.Cluster)
-		if err != nil {
-			return err
-		}
-		if !ok {
-			return fmt.Errorf("configured cluster %q does not exist", b.Cluster)
-		}
+	err = b.api.CheckRequirements(ctx)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	if err := b.api.CheckRequirements(ctx); err != nil {
		return err
	}
```

Reply:
I hate this pattern. It doesn't make things more readable, wrapping the actual func call into boilerplate error management code, and make it harde to refactor if we later want to change func signature to return a value in addition to `err`. Also, my IDE do collapse all the err management shit as a single line, but won't if I adopt this.